### PR TITLE
Refactor Mech request form, fix formatting issue

### PR DESCRIPTION
**Pull request by Tanya-atatakai**
1) updated `env.example` as it's outdated - now contains all needed envs
2) Removed outer form - not needed anymore
3) in case of an error - show notification instead of the `AlertError` as we should not close the form

| Form - no wallet connected      | Form - wallet connected | Error state | 
| ----------- | ----------- |  ----------- |
| image      | image       | image       |

### CODE REVIEW

File: common-util/List/IpfsHashGenerationModal/HashOfDataFile.jsx [hunk 1]
```diff
@@ -36,52 +35,56 @@ const IpfsModal = ({
...
+              form="ipfsModalForm"
+              htmlType="submit"
+              type="primary"
+              disabled={!account}
+              loading={isHashLoading || isLoading}
+            >
+              Request
+            </Button>
+          </div>
+          {!account && <div className="text-gray-500 mt-12">* Please connect your wallet</div>}
```
Comment (oaksprout) [inline common-util/List/IpfsHashGenerationModal/HashOfDataFile.jsx:87]:
```suggestion
          {!account && <div className="text-gray-500 mt-12">To make a request, connect your wallet</div>}
```

Comment (oaksprout) [reply common-util/List/IpfsHashGenerationModal/HashOfDataFile.jsx:87]: Generally good to start the error with the name of the task the user is trying to achieve, followed by what they can do to fix.

**Review by oaksprout — DISMISSED** [review]
image

Happy path looks good.

Small usability comment.

File: common-util/List/IpfsHashGenerationModal/HashOfDataFile.jsx (hunk 1 above)
Comment (Tanya-atatakai) [reply common-util/List/IpfsHashGenerationModal/HashOfDataFile.jsx]: makes sense, fixed

**Review by oaksprout — APPROVED** [review]

**Review by truemiller — APPROVED** [review]

**Pull request merged**